Source/Modules/go.cxx

```diff
@@ -328,11 +335,30 @@ class GO:public Language {
     Printf(go_filename, "%s%s.go", SWIG_output_directory(), module);
 
     String *gc_filename = NULL;
+    String *gc_go_filename = NULL;
     if (!gccgo_flag) {
       gc_filename = NewString("");
+      gc_go_filename = NewString("");
       Printf(gc_filename, "%s%s_gc.c", SWIG_output_directory(), module);
+      Printf(gc_go_filename, "%s%s_gc.go", SWIG_output_directory(), module);
     }
 
+    // Generate unique id based on hash of SWIG input
+
+    void siphash(uint64_t *out, const char *in, uint64_t inlen);
```

Comment:
Make siphash static.  Move the function closer to the top of the file so that it doesn't need a forward declaration.